sympy/solvers/tests/test_ode.py

```diff
@@ -3595,9 +3596,17 @@ def test_factorable_series():
           *(2 - sqrt(3)))*(-sqrt(3) + 1 + (3 - sqrt(3))*(4 - sqrt(3)))) - x**2/(-sqrt(3) - 1
           + (1 - sqrt(3))*(2 - sqrt(3))) + 1) + O(x**6))]
     assert set(sol) == set(dsolve(eq, f(x), hint='factorable'))
-    # checkodesol doesn't work with series solution.
     # FIXME: assert checkodesol(eq, sols) == 2*[(True, 0)]
 
+    eq = x**2*f(x)**2 - 2*x*f(x)*Derivative(f(x), (x, 2)) + Derivative(f(x), (x, 2))**2
+    sol = Eq(f(x), C2*(x**3/6 + 1) + C1*x*(x**3/12 + 1) + O(x**6))
```

Comment:
Can we do the same as a 1st order equation like
```
    eq = x**2*f(x)**2 - 2*x*f(x)*Derivative(f(x), x) + Derivative(f(x), x)**2
```
It would be good if we could use one that doesn't have a series solution instead and that wouldn't need to be a slow test.